lib/travis/build/script/erlang.rb

```diff
@@ -24,7 +24,12 @@ def install
             sh.cmd './rebar get-deps', fold: 'install', retry: true
           end
           sh.elif rebar_configured do
-            sh.cmd 'rebar get-deps', fold: 'install', retry: true
+            sh.if "command -v rebar3" do
+              sh.cmd 'rebar3 get-deps', fold: 'install', retry: true
```

Comment:
Actually, in rebar3 this command has become obsolete. You can [just call `rebar3 compile`](https://www.rebar3.org/v3/docs/basic-usage#section-building) (which is already done below):